src/server/system_services/account_server.js

```diff
@@ -299,6 +299,48 @@ async function generate_account_keys(req) {
     });
 }
 
+/**
+ *
+ * UPDATE_ACCOUNT_KEYS
+ *
+ */
+async function update_account_keys(req) {
+    const account = system_store.get_account_by_email(req.rpc_params.email);
+    if (!account) {
+        throw new RpcError('NO_SUCH_ACCOUNT', 'No such account email: ' + req.rpc_params.email);
+    }
+    if (req.system && req.account) {
+        if (!is_support_or_admin_or_me(req.system, req.account, account)) {
+            throw new RpcError('UNAUTHORIZED', 'Cannot update account');
```

Comment:
Maybe give more details in the message - why "Cannot update account"?

Reply:
The logic for this msg here is that the requesting account is none of the authorized user(support, admin or the user itself). So we are simply saying that `UNAUTHORIZED`. I don't think we need to put any additional details here.